src/library_gl.js

```diff
@@ -1703,7 +1703,7 @@ var LibraryGL = {
 #if GL_ASSERTIONS
     GL.validateGLObjectID(GL.textures, texture, 'glBindTexture', 'texture');
 #endif
-    GLctx.bindTexture(target, texture ? GL.textures[texture] : null);
```

Comment:
Nice to rely on `textures[0] === null`!

Please add a comment in the source where we define those arrays, that index 0 contains null for that purpose (reduce the risk of later confusion).

Reply:
There is already a comment `counter: 1, // 0 is reserved as 'null' in gl` that has been around for a long time it looks like.